xml/depl_maintenance.xml

```diff
@@ -508,6 +508,277 @@
+      </para>
+<screen>crm resource stop apache2 keystone cinder-api glance-api \
+      neutron-server swift-proxy nova-api magnum-api sahara-api heat-api</screen>   
+      <note>
+       <para>
+	If the <literal>Manage stateless active/active services with
+	Pacemaker</literal> option in the &pacemaker; &barcl; is set to
+	<literal>false</literal>, the &ostack; services must be stopped on each
+	node using the <command>systemctl</command> command.
+       </para>
+      </note>
+      <para>
+       <emphasis>From this point, &ostack; services
+       become unavailable.</emphasis>
+      </para>
+     </step>
+     <step>
+      <para>
+       You can now migrate databases using the psql2mysql tool. To upgrade a
+       single database, use the following command format:
+      </para>
+<screen>&prompt.user;psql2mysql \
+--source postgresql://neutron:secret@192.168.1.1/neutron \
+--target mysql+pymysql://neutron:evenmoresecret@192.168.1.2/neutron?charset=utf8 \
+migrate</screen>
+      <para>
+       To migrate all databases in one operation, use the
+       <literal>--batch</literal> option and the
+       <filename>/etc/pg2mysql/databases.yaml</filename> file as follows:
+      </para>
+<screen>&prompt.user;psql2mysql.py --batch /etc/pg2mysql/databases.yaml migrate</screen>
+     </step>
+     <step>
+      <para>
+       In the &crow; &wi;, switch to the <guimenu>Database</guimenu>
+       &barcl;. Enable the raw view and set the value of
+       <literal>sql_engine</literal> to <literal>mysql</literal>. Apply the
+       &barcl;. After this step, &ostack; services should be running again and
+       reconfigured to use the &mariadb; database back end.
+      </para>
+     </step>
+     <step>
+      <para>
+        To prevent the &postgres;-related chef code from running, unassign the values
+	from <literal>database-server</literal> role in the <guimenu>Database</guimenu>
+       &barcl;, and apply the &barcl;.
+      </para>
+     </step>
+     <step>
+      <para>
+       Start chef-client services on the nodes again.
+      </para>
+     </step>
+     <step>
+      <para>
+       Stop &postgres; on the &dbnode;s. Uninstall &postgres; packages.
+      </para>
+      <substeps>
+       <step>
+	<para>
+	 To stop the <literal>postgresql</literal> service, run the following
+	 command on one cluster node:
+	</para>
+<screen>&prompt.user;crm resource stop postgresql
+&prompt.user;crm resource stop fs-postgresql
+&prompt.user;crm resource stop drbd-postgresql</screen>
+        <para>
+	 Run the last command only if the previous setup used DRBD.
+	</para>
+       </step>
+       <step>
+	<para>
+	 Remove the packages on all cluster nodes:
+	</para>
+	<screen>zypper rm postgresql94 postgresql94-server</screen>
+       </step>
+       <step>
+	<para>
+	 If you choose not to upgrade to &productname; 8 right away, delete unused pacemaker resource from one cluster node:
+	</para>
+<screen>&prompt.user;crm conf delete drbd-postgresql
```

Comment:
Do we need to make a note that you only need to delete this drbd resource if the preious setup used drbd (because we pointed it out in 10a?